ui/components/app/signature-request-siwe/signature-request-siwe.js

```diff
@@ -0,0 +1,150 @@
+import React, { useContext, useState } from 'react';
+import PropTypes from 'prop-types';
+import convertMsg from '../../../helpers/utils/format-message-params';
```

Comment:
I recommend avoiding synonyms. We should keep naming consistent. Can change `convertMsg` to `formatMessageParams` to match the filename?

Reply:
Done!